sacc/sacc.py

```diff
@@ -371,16 +371,18 @@ def indices(self, data_type=None, tracers=None, warn_empty=True, **select):
             # Record this index
             if ok:
                 indices.append(i)
+
         if len(indices) == 0 and warn_empty:
             if tracers is None:
                 warnings.warn("Empty index selected")
             else:
                 warnings.warn("Empty index selected - maybe you "
                               "should check the tracer order?")
+
         return np.array(indices, dtype=int)
 
     def remove_selection(self, data_type=None, tracers=None,
-                         warn_empty=True, **select):
+                         warn_empty=False, **select):
```

Comment:
Why do we prefer False here? I did find these warnings useful (no biggie if we can turn then back on, but just in case)

Reply:
It was showing up a lot when doing loops where some pairs didn't exist, but probably I should just have been pasing `warn_empty=False`, you're right.  I'll switch this back!